docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -39,70 +47,188 @@ Use this command to switch to the cluster assigned to `CONTEXT_NAME`:
-  kube_public_addr: kube.example.com:3026
+Run the `get-kubeconfig.sh` script:
 
-kubernetes_service:
-  enabled: yes
-  listen_addr: 0.0.0.0:3027
-  # replace this path with the actual path for your generated kubeconfig
-  kubeconfig_file: "path/to/kubeconfig"
+```code
+$ bash get-kubeconfig.sh
+```
+
+The script is successful if you see this message:
+
+```text
+Done!
+```
+
+Move the `kubeconfig` to the host you are using to run the Teleport Kubernetes
+Service. We will assume that the `kubeconfig` file exists at
+`/var/lib/teleport/kubeconfig`.
+
+<Details title="Connecting multiple Kubernetes clusters?">
+
+You can connect multiple Kubernetes clusters to Teleport from one `kubeconfig`
+if it contains multiple entries. Use
+[merge-kubeconfigs.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/merge-kubeconfigs.sh)
+to combine multiple `kubeconfigs` generated by `get-kubeconfig.sh`.
+
+</Details>
+
+## Step 2/4. Set up the Teleport Kubernetes Service
+
+In this step, you will install the Teleport Kubernetes Service and configure it
+to use the `kubeconfig` you generated to access a Kubernetes cluster.
+
+### Get a join token
+
+Establish trust between your Teleport cluster and your new Kubernetes Service
+instance by creating a join token:
+
+```code
+$ tctl nodes add --roles=kube
+The invite token: (=presets.tokens.first=)
+This token will expire in 30 minutes.
+Run this on the new node to join the cluster:
+> teleport start \
+   --roles=kube \
+   --token=(=presets.tokens.first=) \
+   --ca-pin=(=presets.ca_pin=) \
+   --auth-server=192.0.2.255:3025
+Please note:
+  - This invitation token will expire in 30 minutes
+  - 192.0.2.255:3025 must be reachable from the new node
 ```
 
+On the host where you are running the Teleport Kubernetes Service, create a
+file called `/tmp/token` that consists only of your token.
+
+```code
+$ echo <Var name="join-token" /> | sudo tee /tmp/token
+```
+
+### Install the Teleport Kubernetes Service
+
+Run the following commands on the host where you will install the Teleport
+Kubernetes Service:
+
+<Tabs dropdownView dropdownCaption="Teleport Edition">
+<TabItem label="Self-Hosted" scope={["oss", "enterprise"]}>
+
+(!docs/pages/includes/install-linux.mdx!)
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope="cloud">
+
+(!docs/pages/includes/install-agent-cloud.mdx service="the Teleport Kubernetes Service"!)
+
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Cloud">
+</Tabs>
+
+### Configure the Teleport Kubernetes Service
 
-On the host you will use to run the Teleport Kubernetes Service, add the
-following to `teleport.yaml`.
+On the host where you will run the Teleport Kubernetes Service, create a file at
+`/etc/teleport.yaml` with the following content:
 
 ```yaml
-# ...
+version: v2
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  auth_servers:
+  - "teleport.example.com:3080"
```

Comment:
Can/should we update that to use v3 of the config, replacing `auth_servers` with `proxy_server`?